client/src/redux/sagas/datafiles.sagas.js

```diff
@@ -317,7 +317,7 @@ export function* watchMove() {
   yield takeLeading('DATA_FILES_MOVE', moveFiles);
 }
 
-export function* moveFile(src, dest, index) {
+export function* moveFile(src, dest, index, keepModalOpen = false) {
```

Comment:
What's the purpose of the `keepModalOpen` param?

Reply:
That param is there for the outer `moveFiles` function to read in the event there are multiple files being moved - this was my solution for keeping the modal open if any of the individual `moveFile` operations fail. `keepModalOpen` is, by default, `false`, and will only switch to `true` if an error is thrown in `moveFile`. This is the same for `copyFile` and `trashFile`.

Reply:
Can you point me to where it's used by `moveFiles`? I'm not seeing that

Reply:
I see. Ok, I'll wait for that PR before reviewing further